Description: Fixed UnboundLocalError
Author:  Geoffroy Youri Berret <efrim@azylum.org>
Applied-Upstream: https://github.com/nicolargo/glances/commit/fc1b009
Last-Update: 2012-12-30
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/glances/glances.py
+++ b/glances/glances.py
@@ -1787,10 +1787,10 @@
                 format(self.__autoUnit(mem['free']), '>5'), 5)
 
             # Display extended informations if space is available
+            y = 0
             if screen_x > self.mem_x + offset_x + memblocksize:
                 # active and inactive (only available for psutil >= 0.6)
                 if psutil_mem_vm:
-                    y = 0
                     # active
                     if 'active' in mem:
                         self.term_window.addnstr(self.mem_y + y,
